pkg/granted/tokens.go

```diff
@@ -111,8 +120,21 @@ var TokenExpiryCommand = cli.Command{
 				expiry = token.Expiry.Local().Format(time.RFC3339)
 			}
 
-			clio.Logf("%-*s (%s) expires at: %s", max, key, strings.Join(startUrlMap[key], ", "), expiry)
+			sso_expiry_data := sso_expiry{
```

Comment:
@shwethaumashanker could you make it return JSON only if `--output=json` is set? 

I am thinking that we can use `--output=json` as a consistent flag in our CLIs to return JSON, otherwise we can print a more human-readable table output

Reply:
I added `--json` flag since  `--output=json` was throwing a error: 
<img width="952" alt="Screenshot 2023-11-08 at 5 27 41 PM" src="https://github.com/common-fate/granted/assets/32020525/915c282f-b467-4e3f-b2df-b86f16c57b19">

Let me know if this is okay or I can figure out a way to add `--output=json`